Tasks/DeployVisualStudioTestAgent/TestAgentConfiguration.ps1

```diff
@@ -15,18 +14,35 @@
     )
 
     Try {
+
+        $Domain = "."
+        $TestUser = ""
+        $DomainUser = $TestUserName.Split("\")
```

Comment:
Do we need to check if $TestUserName is null or empty? Or is this validation done before?

Reply:
This validation is done upfront.